Governance/tech/conventions.md

```diff
@@ -1,147 +1,295 @@
-# Applying conventions to your files
+useful summary: https://cfconventions.org/Data/cf-documents/requirements-recommendations/conformance-1.9.html
+# Writing valid and compliants netCDF files
 
+This page covers the basic netCDF structure and the application of the CF and ACDD conventions to netCDF files. Not all the rules are listed here, this is just a summary of the most common ones, for a complete overview of both conventions refer to their official documentation (see references).
 
 ## Naming conventions
 
+There are no recognised vocabularies for files and variable names, however the ESGF created their own [variable definition tables](https://github.com/PCMDI/cmip6-cmor-tables/tree/master/Tables) for the CMIP output and this is often used by other projects.
+The CF conventions only have some basic requirements and recommendations on how to form the names. 
 
+**Requirements**
+
+* NetCDF files are required to have the file name extension ".nc".
```

Comment:
is ".nc4" valid?

Reply:
I can't find any reference to 'nc4' in the conventions document, both cf conventions and unidata (https://docs.unidata.ucar.edu/netcdf-c/current/faq.html#What-convention-should-be-used-for-the-names-of-netCDF-files) list ".nc" as correct. However I'm sure that other suffixes or no suffix still work, but might cause issues with particular softwares.